src/screens/OrganizationPeople/OrganizationPeople.test.tsx

```diff
@@ -493,24 +509,35 @@ describe('Organization People Page', () => {
         rowsPerPageSelect?.querySelectorAll('option')
       );
 
-      const peopleListContainer = screen.getByTestId('orgpeoplelist');
-
       // Change the selected option of dropdown to the value of the current option
       userEvent.selectOptions(
         rowsPerPageSelect,
         rowsPerPageOptions[currRowPPindex].textContent
       );
 
-      const totalNumPeople =
-        rowsPerPageOptions[currRowPPindex].textContent === 'All'
-          ? getTotalNumPeople(allPeopleTypes[peopleTypeIndex])
-          : parseInt(rowsPerPageOptions[currRowPPindex].value);
-
-      expect(
-        Array.from(
-          peopleListContainer.querySelectorAll('[data-testid="peoplelistitem"]')
-        ).length
-      ).toBe(totalNumPeople);
+      const totalNumPeople = MOCKS[3]?.result?.data?.users?.filter(
+        (datas: {
+          _id: string;
+          lastName: string;
+          firstName: string;
+          image: string;
+          email: string;
+          createdAt: string;
+          joinedOrganizations: {
+            __typename: string;
+            _id: string;
+          }[];
+        }) => {
+          window.location.assign('/orgpeople/id=6401ff65ce8e8406b8f07af1');
+          const pathname = window.location.pathname;
+          const id = pathname.split('=')[1];
+          return datas.joinedOrganizations.some((org) => org._id === id);
+        }
+      ).length;
+
+      await wait();
+      const expectedUsersLength = screen.getAllByTestId('orgpeoplelist').length;
+      expect(expectedUsersLength).toBe(totalNumPeople);
```

Comment:
`expectedUsersLength` is a variable to store the amount of users rendered in the test. The variable `totalNumPeople` would be best to switch what it stores with `expectedUsersLength`. That way the code reasons

`expect(totalNumPeople).toBe(expectedUsersLength)`

Reply:
@Rumone Thanks for review... I have implemented the changes..